benchmarks/100.webapps/120.uploader/input.py

```
@@ -1,19 +1,25 @@
+
 def buckets_count():
     return (0, 1)
 
-def generate_input(data_dir, size, benchmarks_bucket, input_buckets, output_buckets, upload_func, nosql_func):
-    input_config = {'object': {}, 'bucket': {}}
-    input_config['object']['url'] = url_generators[size]
-    input_config['bucket']['bucket'] = benchmarks_bucket
-    input_config['bucket']['output'] = output_buckets[0]
+
+def generate_input(
+    data_dir, size, benchmarks_bucket, input_buckets, output_buckets, upload_func, nosql_func
```

Ruff (0.14.5)

Unused function argument: `data_dir`

(ARG001)